packages/framework-integration-tests/src/config/config.ts

```diff
@@ -28,4 +28,8 @@ Booster.configure('production', (config: BoosterConfig): void => {
 
   config.appName = 'my-store-' + appNameSuffix
   config.provider = AWS.Provider()
+  config.tokenVerifier = {
+    jwksUri: 'https://booster-integration-tests.s3.amazonaws.com/.well-known/jkws.json',
```

Comment:
Depending on an external resource in a hardcoded URL scares me... maybe we could generate it dynamically, it's a little more work now, but it could save a lot of headaches in the future...